src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceKind.kt

```diff
@@ -19,4 +19,10 @@ enum class LicenceKind(
     LicenceEventType.VARIATION_CREATED,
     LicenceEventType.VARIATION_SUBMITTED,
   ),
+
+  HARDSTOP(
+    IN_PROGRESS,
+    LicenceEventType.HARDSTOP_CREATED,
+    LicenceEventType.HARDSTOP_SUBMITTED,
+  ),
```

Comment:
Its not necessarily a code change request just I am only thinking aloud. my feeling is that draft licence workflow/state might be generic, like create, amend, submit, approve, then separately it might be status of active, inactive irrespective of what type of licence it is. I appreciate some types of draft licence might have their unique states and rules as well.

Reply:
Yes, think the workflow states are pretty generic for licences, but think its worth being able to measure or track events specific to the type of licence for auditing purposes - tells a more compelling story when looking across licences for a given prisoner"
```
LICENCE_CREATED
LICENCE_TIMEDOUT
HARDSTOP_CREATED
HARDSTOP_SUBMITTED
LICENCE_ACTIVATED
VARIATION_CREATION
VARIATION_ACTIVATED
...
```